Config/Framework.xcconfig

```diff
@@ -0,0 +1 @@
+CLANG_MODULES_AUTOLINK = NO
```

Comment:
This should be added to the library scheme but not the framework scheme, when the framework is built it  takes the lib scheme.

Reply:
I really don't know why it did work. I made a change to make it work.